blocks/explore-articles/explore-articles.js

```diff
@@ -1,20 +1,21 @@
-import { createElement, getTextLabel } from '../../scripts/common.js';
+import { createElement, getTextLabel, capitalizeWords } from '../../scripts/common.js';
 import {
-  fetchMagazineArticles,
-  getArticleTagsJSON,
-  removeArticlesWithNoImage,
+  fetchMagazineData,
+  formatArticlesArray,
+  formatFacetsArray,
 } from '../../scripts/services/magazine.service.js';
 
-const allArticles = await fetchMagazineArticles();
-const allArticlesWithImage = removeArticlesWithNoImage(allArticles);
-const allArticleTags = await getArticleTagsJSON();
-const allCategories = allArticleTags.categories;
-const allTrucks = allArticleTags.trucks;
 const blockName = 'explore-articles';
 
+const queryVariables = { facets: ['ARTICLE', 'TRUCK'], sort: 'LAST_MODIFIED_DESC' };
+const allData = await fetchMagazineData(queryVariables);
+const allArticles = formatArticlesArray(allData?.items);
+const allFacets = formatFacetsArray(allData?.facets);
+
+const { truck: allTrucks, category: allCategories } = allFacets;
 const [categoryPlaceholder, truckPlaceholder] = getTextLabel('Article filter placeholder').split(',');
 
-const articlesPerChunk = 4;
+const artsPerChunk = 4;
```

Comment:
why do you reduce the name to a reduce one?

Reply:
I had a `max-length` linter issue so I made this var name shorter 🤷